docs/spec.md

```diff
@@ -1135,85 +1136,252 @@ in Evy. The call arguments must be listed individually.
 
 ## Break and Return
 
-`break` and `return` are terminating statements. They interrupt the
-regular flow of control. `break` is used to exit from the inner-most
-loop body. `return` is used to exit from a function and may be followed
-by an expression whose value is returned by the function call.
+`break` and `return` are _terminating statements_ in Evy. They interrupt
+the regular flow of control.
+
+- `break` is used to exit from the innermost loop body. This means that
+  it will skip the rest of the loop body and continue with the next
+  statement after the loop.
+- `return` is used to exit from a function. It can be followed by an
+  expression whose value is returned by the function call.
+
+For example, the following code shows how the `break` statement can be
+used to exit from a loop:
+
+```evy
+for x := range 2
+    y := 0
+    while y < 10
+        if y == 2
+            print "break" y
+            break
+        end
+        print "no break" y
+        y = y + 1
+    end
+    print "x" x "y" y
+    print
+end
+```
+
+This code will print the following output:
+
+```evy:output
+no break 0
+no break 1
+break 2
+x 0 y 2
+
+no break 0
+no break 1
+break 2
+x 1 y 2
+
+```
+
+As you can see, the `break` statement causes the loop to exit when the
+value of `y` is equal to 2. The next statement after the loop is then
+executed.
+
+The following code shows how the return statement can be used to exit
+from a function:
+
+```evy
+func foo:string
+    if (rand1) < 0.7
+        return "bar"
+    else
+        return "baz"
+    end
+end
+```
+
+This code will return the value of `"bar"` 70% of the time and `"baz"`
+otherwise. The names [foo, bar, and baz] are common placeholder names
+used in code.
+
+[foo, bar, and baz]: https://en.wikipedia.org/wiki/Foobar
 
 ## Typeof
 
-`typeof` returns the concrete type of a value held by a variable as a
-string. It returns a string the same as the type in an evy program,
-e.g. `num`, `bool`, `string`, `[]num`, `{}[]any`, etc. For an empty
-composite literal , `typeof` returns `[]` or `{}` as it can be matched
-to any subtype. e.g. `[]` can be passed to a function that takes an
-argument of `[]num`, or `[]string`, etc.
+The `typeof` function returns the concrete type of a value held by a
+variable as a string. It returns a string that is the same as the type
+in an Evy program, such as `"num"`, `"bool"`, `"string"`, `"[]num"`, `"
```

Comment:
putting a line break within the backticks causes a space to go into the text (`" {}[]any"`). I would move the line break before the opening backtick if you can - not sure if prettier is going to just move that again though.

Reply:
sorry, fixed.